CONTRIBUTING.md

```diff
@@ -16,6 +16,8 @@ You can do this online, and it only takes a minute.
 
 2. Understand [coding agreements](packages/README.md).
 
+3. **[Optional]** Follow instructions to [enable and activate DCM (Dart Code Metrics)](#enabling-and-activating-dcm).
```

Comment:
Lets remove #3 here and make the entire section to be one more subsection of 
`Development prep`.

Reply:
Done